hack/ginkgo-e2e.sh

```diff
@@ -148,6 +148,7 @@ export PATH=$(dirname "${e2e_test}"):"${PATH}"
   --network="${KUBE_GCE_NETWORK:-${KUBE_GKE_NETWORK:-e2e}}" \
   --node-tag="${NODE_TAG:-}" \
   --master-tag="${MASTER_TAG:-}" \
+  --cluster-monitoring="${KUBE_ENABLE_CLUSTER_MONITORING:-influxdb}" \
```

Comment:
I wouldn't push on it - but `cluster-monitoring-mode` might be a better representation of what the flag is doing.